packages/perseus/src/server-item-renderer.tsx

```diff
@@ -136,6 +139,16 @@ export class ServerItemRenderer
                 this.props.onRendered(true);
             }
         }
+
+        if (this.props.score && this.props.score !== prevProps.score) {
```

Comment:
**question**: Should we use a deep equality check here instead of simply instance comparison? 🤔  If the score is identical, I guess it could still mean that there are different widgets that are empty and need to be highlighted.  I think I answered my own question. 😖